docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -592,6 +592,14 @@ apply `optimize` only on the partitions corresponding to the filter:
 ALTER TABLE test_partitioned_table EXECUTE optimize
 WHERE partition_key = 1
 ```
+You can use a more complex `WHERE` to narrow down the scope of the 
+`optimize` procedure. The following example casts the data to a date and uses
+a comparison to only optimize partitions with data from the year 2022 or newer.
```

Comment:
You changed the sentence to the old state. Please revert and fix only backtick number.